manual/Website2/src/components/Title/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-// import 'moment'/
+import 'react-select';
```

Comment:
Added to break importing title component from website2 to website1 - forces to ensure dependency chain loading works